build-tools/packages/build-cli/src/test/machines/fluidReleaseMachine.test.ts

```diff
@@ -137,12 +138,14 @@ describe("FluidReleaseMachine", () => {
 
 			if (!state.startsWith("Do") || requiresBothActions.has(state)) {
 				it(state, () => {
-					expect(exits).to.be.equalTo(["failure", "success"]);
+					// eslint-disable-next-line unicorn/no-array-for-each
+					["failure", "success"].forEach((item) => expect(exits).to.be.containing(item));
```

Comment:
Does the order really not matter here?

Reply:
No, but I was able to simplify/clarify these tests.